### DISCUSSION TITLE
Daly BMS showing massively oscillating currents

### DISCUSSION THREAD

**Discussion by floriangramer** [Louisvdw/dbus-serialbattery#739] [Q&A] [answered]
Hi there! I am using a Daly BMS (16s/100A/LiFePo4) connected via the original daly UART-cable to a Raspberry Pi running Venus.OS. When the Multiplus is inverting at a power of about 600W the currents in the console @ the battery are displayed as oscillating at a frequency of about 0.5Hz between 5A and 23A. Luckily this current doesn't get used by the control algorithms as the actual current drawn by the Multiplus remains stable and only changes to correct for changes in the power drawn. It looks strange, though. Any ideas as to how I could fix this?
Btw.: I've tried two different BMSes, two different Adapters, and even different versions of the driver and I always seem to get this behavior.

**Comment by drtinaz** [comment, marked as answer]
This has nothing to do with the driver. It is an issue with the bms and not being able to accurately measure the current. Unfortunately a common theme with Daly bms's. If you connect the uart directly to a pc and monitor it via Daly bms software you will see the same thing.

**Reply by floriangramer** [reply]
Thanks very much - would have been easy enough to find if I had only searched correctly. I am going to contact the seller and demand a fix (be it another firmware or a replacement).
